packages/blade/src/components/Icons/_Svg/Svg/types.ts

```diff
@@ -9,4 +9,4 @@ export type SvgProps = {
   width: string;
   fillOpacity?: number;
 } & TestID &
-  Omit<StyledPropsBlade, 'order'>; // Order prop on SVG has different meaning so removing this prop from styledProps
+  Omit<StyledPropsBlade, 'order' | 'visibility'>; // Order prop on SVG has different meaning so removing this prop from styledProps
```

Comment:
fyi visibility prop is getting into DOM: 

<img width="215" alt="image" src="https://github.com/razorpay/blade/assets/35374649/518917c1-f20e-4436-9736-353ade5ba939">

Reply:
Use a transient prop `$visibility`?

Reply:
We have a filter on Box to filter out some of these props. I added visibility there so it should be fixed now